src/3rd_party/cnpy/cnpy.h

```diff
@@ -5,7 +5,9 @@
 #ifndef LIBCNPY_H_
 #define LIBCNPY_H_
 
+#if !defined(WASM)
```

Comment:
Fun fact: now that @XapaJIaMnu has binary working, you can get rid of cnpy entirely.

Reply:
Yes, but for compilation we will still have to keep this ifdef. Right?